### PR TITLE
AN-7880 UserActivityTask: process new forum event_types

### PR DESCRIPTION
**Pull request by thallada**
This will enable processing forum events in the weekly course activity task. I tested this locally with some (slightly modified) [example forum events from student_forum_activity.log](https://github.com/edx/edx-analytics-pipeline/blob/master/edx/analytics/tasks/tests/acceptance/fixtures/input/student_forum_activity.log) in my tracking.log.

Some things I wasn't able to confirm:
- If these event_types are really what the forums emits now (I can't get forums working on my devstack)
- ~~If I'm missing or including and event I shouldn't in DISCUSSION_ACTIVITIES. It depends on how we want to define what a "discussion activity" is.~~ Nevermind, I talked to Katy and now I'm copying what the learner roster considers "discussion activity"

### CODE REVIEW

File: edx/analytics/tasks/tests/test_user_activity.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def test_problem_event(self):
         self.assertEquals(event, expected)
 
     def test_post_forum_event(self):
-        line = self.create_event_log_line(event_source='server', event_type='blah/blah/threads/create')
+        line = self.create_event_log_line(event_source='server', event_type='edx.forum.thread.created')
```
Comment (mulby) [inline edx/analytics/tasks/tests/test_user_activity.py]: given the logic in the mapper, I'm assuming there are other types of events that should be counted as well? Can you enumerate the ones we know about here?

Comment (thallada) [reply edx/analytics/tasks/tests/test_user_activity.py]: Good point, I'll test those events too.

**Review by dsjen — APPROVED** [review]
Nice to see!  LGTM, although I don't know much about the pipeline...yet.  👍

File: edx/analytics/tasks/tests/test_user_activity.py [hunk 2]
```diff
@@ -113,12 +113,33 @@ def test_problem_event(self):
...
         event = tuple(self.task.mapper(line))
         expected = (((self.course_id, self.username, self.expected_date_string, ACTIVE_LABEL), 1),
                     ((self.course_id, self.username, self.expected_date_string, POST_FORUM_LABEL), 1))
         self.assertEquals(event, expected)
 
+    def test_forum_response_event(self):
```
Comment (mulby) [inline edx/analytics/tasks/tests/test_user_activity.py]: I think we could make this more DRY with ddt!

Comment (thallada) [reply edx/analytics/tasks/tests/test_user_activity.py]: I've never used ddt before... it's pretty convenient!

**Comment by thallada** [comment]
Can someone on the pipeline team give this a review / thumbs up when they get a chance?

**Review by brianhw — APPROVED** [review]
LGTM.  👍    Thanks, Tyler!

**Comment by thallada** [comment]
Thanks Brian! :ship:

**Pull request merged**